articles/stay-on-course-with-your-security-compliance-goals.md

```diff
@@ -0,0 +1,54 @@
+# Stay on course with your security compliance goals
+
+Pursuing security compliance is a long journey. It isn’t so much taking a road trip as it’s setting sail on the open ocean. Regulatory changes can knock you off course. Learning a new technology can take the wind out of your sails. And no matter how hard you try, you can never reach the horizon.
+
+Are you feeling a little seasick? Don’t worry. There are ways to navigate the waters of security compliance and still enjoy smooth sailing.
+
+We’ve explored how to [get and stay compliant](https://fleetdm.com/use-cases/get-and-stay-compliant-across-your-devices-with-fleet) before. In this article, we’ll walk through the steps you can take to measure progress toward compliance — and how Fleet can make this task more manageable.
+
+## Set your goals
+
+One of the reasons compliance is so complicated is that there isn’t a single set of rules. You have to determine what compliance means for your organization.
+
+Are you creating internal processes and controls to stay secure? Is there a law or regulation required to do business? Which industry standards should you meet if you truly want to compete?
+
+Answering questions like these is the first step to becoming compliant. After all, you can’t measure progress until you know where you’re going.
+
+## Be realistic
+
+Achieving the highest level of compliance for your industry is an admirable goal. But pursuing compliance perfection could tie up daily business operations.
+
+For instance, if your organization has 30 employees who use MacBooks, it might be possible to make sure they all have the latest version of macOS installed. That goal gets more challenging to achieve as those MacBooks multiply to the hundreds or even thousands. Maybe then you just want to make sure they’re on Big Sur.
```

Comment:
This is an important one to fix. Nobody in their right mind would be OK with people just being on an old OS because they have too many devices. They'll set SLAs, measure against them, and try to prevent access from devices that do not meet them.

Reply:
Will close this conversation as the one below is a duplicate of it anyway.